torch/csrc/autograd/profiler.cpp

```diff
@@ -48,12 +50,16 @@ void mark(std::string name, bool include_cuda /* = true */) {
   }
 }
 
-void pushRangeImpl(const StringView& name, const char* msg="", int64_t sequence_nr=-1) {
+void pushRangeImpl(
+    const StringView& name,
+    const char* msg = "",
+    int64_t sequence_nr = -1,
+    std::vector<std::vector<int64_t>>&& shapes = {}) {
   if (state == ProfilerState::Disabled) {
     return;
   }
   if (state == ProfilerState::NVTX) {
-    if(sequence_nr >= 0) {
+    if (sequence_nr >= 0) {
       std::stringstream s;
       s << name.str() << msg << sequence_nr;
```

Comment:
unrelated to this diff, but isn't msg here always the same constant? can we kill this argument?

Reply:
not sure, I haven't tried using profiler on CUDA. @zdevito  might know